Rakefile

```diff
@@ -216,20 +217,18 @@ namespace :integrations do
     clone_from_git("elixir/integration/appsignal-elixir-plug", "appsignal-elixir-plug")
     # Clone Node.js
     clone_from_git("nodejs/integration", "appsignal-nodejs")
-    reset_repo("nodejs/integration")
     # Clone JavaScript
     clone_from_git("javascript/integration", "appsignal-javascript")
-    reset_repo("javascript/integration")
   end
 
   desc "Reset integrations"
   task :reset do
     # Ruby
     reset_repo("ruby/integration")
     # Elixir
+    reset_repo("elixir/integration/appsignal-elixir")
     reset_repo("elixir/integration/appsignal-elixir-phoenix")
-    reset_repo("elixir/integration/appsignal-phoenix")
-    reset_repo("elixir/integration/appsignal-plug")
+    reset_repo("elixir/integration/appsignal-elixir-plug")
```

Comment:
If we always reset after closing, we could combine the two tasks to clone if the dir is not present and reset if it is. That's what the clone task is doing with your changes. You can remove the reset task then.